.github/workflows/test.yaml

```diff
@@ -32,58 +41,21 @@ jobs:
         java -Djava.library.path=/tmp/DynamoDBLocal_lib -jar /tmp/DynamoDBLocal.jar -inMemory -port 8000 &
 
     - name: Run tests
-      run: |
-        pytest --cov-report term-missing --cov=pynamodb tests
-
-    - name: Upload coverage
-      run: |
-        coveralls --service=github
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        COVERALLS_FLAG_NAME: ${{ matrix.test-name }}
-        COVERALLS_PARALLEL: true
-
-
-  mypy:
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install -e .[signals] -r requirements-dev.txt
-    - name: Run mypy
-      run: |
-        mypy .
-
-  build-docs:
-    runs-on: ubuntu-latest
-
-    steps:
-    - uses: actions/checkout@v2
-    - name: Set up Python 3.8
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install -r docs/requirements.txt
-    - name: Build docs
-      run: |
-        sphinx-build -W docs /tmp/docs-build
-
-  finish:
-    needs: test
-    runs-on: ubuntu-latest
-    steps:
-    - name: Coveralls Finished
-      uses: coverallsapp/github-action@master
-      with:
-        github-token: ${{ secrets.github_token }}
-        parallel-finished: true
+      run: poetry run pytest tests
+#
+#  mypy:
+#    runs-on: ubuntu-latest
+#
+#    steps:
+#    - uses: actions/checkout@v2
+#    - name: Set up Python 3.8
+#      uses: actions/setup-python@v2
+#      with:
+#        python-version: 3.11
+#    - name: Install dependencies
+#      run: |
+#        python -m pip install --upgrade pip
+#        python -m pip install -e .[signals] -r requirements-dev.txt
+#    - name: Run mypy
+#      run: |
+#        mypy .
```

Comment:
Did this not work? I assume the codebase already had working type checking so would be good to keep that if we can...

Reply:
It did, will re-enable